Formula/kde-karchive.rb

```diff
@@ -1,8 +1,9 @@
 class KdeKarchive < Formula
   desc "Reading, creating, and manipulating file archives"
   homepage "https://api.kde.org/frameworks/karchive/html/index.html"
-  url "https://download.kde.org/stable/frameworks/5.76/karchive-5.76.0.tar.xz"
-  sha256 "503d33b247ae24260c73aac2c48601eb4f8be3f10c9149549ea5dd2d22082a2a"
+  url "https://download.kde.org/stable/frameworks/5.77/karchive-5.77.0.tar.xz"
+  sha256 "2239af811b9205d9f4486791f4b25ff5ea299b3873021a646e1c8b9d44c2be36"
+  license all_of: ["BSD-2-Clause", "LGPL-2.0-only", "LGPL-2.0-or-later", "LGPL-3.0-only"]
```

Comment:
The only reason `LGPL-3.0-only` is included is that [`src/karchive.cpp`](https://invent.kde.org/frameworks/karchive/-/blob/master/src/karchive.cpp) contains the identifier `LGPL-2.0-only OR LGPL-3.0-only`. That means that `LGPL-3.0-only` isn't strictly needed. We may want to consider using the following:

```ruby
  license all_of: [
    "BSD-2-Clause",
    "LGPL-2.0-only",
    "LGPL-2.0-or-later",
    any_of: ["LGPL-2.0-only", "LGPL-3.0-only"],
  ]
```